configure/CONFIG

```diff
@@ -1,14 +1,29 @@
-#CONFIG
-include $(TOP)/configure/CONFIG_APP
-# Add any changes to make definitions here
-
-#CROSS_COMPILER_TARGET_ARCHS = vxWorks-68040
-#CROSS_COMPILER_TARGET_ARCHS = 
-
-# Use this when your IOC and the host use different paths
-# to access the application. Typically this will be 
-# used with the Microsoft FTP server or with NFS mounts. Use
-# is indicated by failure of the cdCommands script on
-# vxWorks. You must rebuild in the iocBoot directory 
-# before this takes effect.
-#IOCS_APPL_TOP = <the top of the application as seen by the IOC>
+# CONFIG - Load build configuration data
+#
+# Do not make changes to this file!
+
+# Allow user to override where the build rules come from
+RULES = $(EPICS_BASE)
+
+# RELEASE files point to other application tops
+include $(TOP)/configure/RELEASE
+-include $(TOP)/configure/RELEASE.$(EPICS_HOST_ARCH)
+-include $(TOP)/configure/RELEASE.$(EPICS_HOST_ARCH).Common
+ifdef T_A
```

Comment:
Should CONFIG include those RELEASE.* files? RELEASE does so too. I am not sure what the state-of-the-art method is.

Reply:
At least for asyn, [CONFIG ](https://github.com/epics-modules/asyn/blob/master/configure/CONFIG) includes the RELEASE and RELEASE*.config files, and [RELEASE](https://github.com/epics-modules/asyn/blob/master/configure/RELEASE) includes the RELEASE*.local files, which I believe is what I've setup up here.

This seems to be the case for other modules as well such as [ipac](https://github.com/epics-modules/ipac/tree/master/configure), [sscan](https://github.com/epics-modules/sscan/tree/master/configure), [autosave ](https://github.com/epics-modules/autosave/tree/master/configure)and others.